airflow-core/src/airflow/ui/src/layouts/Details/Grid/GridTI.tsx

```diff
@@ -106,35 +109,44 @@ const Instance = ({ dagId, instance, isGroup, isMapped, onClick, runId, taskId }
       py={0}
       transition="background-color 0.2s"
     >
-      <Link
-        id={`grid-${runId}-${taskId}`}
-        onClick={onClick}
-        replace
-        to={{
-          pathname: getTaskUrl(),
-          search: redirectionSearch,
-        }}
+      <BasicTooltip
+        content={
+          <>
+            {translate("taskId")}: {taskId}
+            <br />
+            {translate("state")}:{" "}
+            {instance.state
+              ? translate(`common:states.${instance.state}`)
+              : translate("common:states.no_status")}
+            {instance.min_start_date !== null && (
+              <>
+                <br />
+                {translate("startDate")}: <Time datetime={instance.min_start_date} />
+              </>
+            )}
+            {instance.max_end_date !== null && (
+              <>
+                <br />
+                {translate("endDate")}: <Time datetime={instance.max_end_date} />
+              </>
+            )}
+            {Boolean(duration) && (
+              <>
+                <br />
+                {translate("duration")}: {duration}
+              </>
+            )}
+          </>
+        }
```

Comment:
We should move this to its own PR. Because this will only work for task group, if we want this everywhere, we need to discuss if we want to update the backend TiSummary, or not. 

In any case I think it's unrelated to this PR, and to not make the scope bigger I would split it.